cli/module_build.go

```diff
@@ -490,6 +491,68 @@ type reloadModuleArgs struct {
 	NoBuild     bool
 	Local       bool
 	NoProgress  bool
+	CloudBuild  bool
+	BuildID     string
+	CloudConfig string
```

Comment:
CloudConfig is confusing imo. Can you drop a comment saying this is the json config containing the robot ID? (I know we use this name in rdk as well)